apisix/schema_def.lua

```diff
@@ -642,6 +642,18 @@ _M.ssl = {
             type = "array",
             items = private_key_schema,
         },
+        client = {
+            type = "object",
+            properties = {
+                ca = certificate_scheme,
```

Comment:
We may support CA cert chains in the future.

Reply:
Currently, OpenResty only supports a single CA cert chain. We can use `ca_list` or anything else if multiple chains are supported. However, using such a name now is confusing as we only support one.

Reply:
OK, just an advice and won't block this PR.